### PR TITLE
Rca issue tracker

### PR DESCRIPTION
**Pull request by ChadNedzlek**
Respond to the "issues" web hook to implement our RCA issue tracking workflow.

When an issue with a particular tag (currently "RCA Requested") is closed, open a new issue
and port a few tags and information over.

### CODE REVIEW

File: src/DotNet.Status.Web/appsettings.json [hunk 1]
```diff
@@ -41,9 +41,14 @@
     "TableUri": "<<must override in environment>>",
     "TableSasToken": "[vault(deployment-table-sas-token)]"
   },
+  "WebHooks": {
     "github": {
       "SecretKey": {
-  } 
+        "default": "[vault(github-webhook-secret)]"
```
Comment (garath) [inline src/DotNet.Status.Web/appsettings.json]: This doesn't yet exist in the staging or prod vaults

Comment (ChadNedzlek) [reply src/DotNet.Status.Web/appsettings.json]: Oh! Correct. I haven't turned on the webhook yet.

Comment (ChadNedzlek) [reply src/DotNet.Status.Web/appsettings.json]: Added. :-)

Comment (ChadNedzlek) [reply src/DotNet.Status.Web/appsettings.json]: Turned it on and popped them in there.

**Comment by ChadNedzlek** [comment]
I'm merging!

**Pull request merged**